src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -118,12 +118,18 @@ export class CalciteAlert {
     if (!scale.includes(this.scale)) this.scale = "m";
 
     let durations = ["slow", "medium", "fast"];
+
     if (
       this.autoDismissDuration !== null &&
       !durations.includes(this.autoDismissDuration)
     ) {
       this.autoDismissDuration = "medium";
     }
+
+    if (this.active) {
```

Comment:
Shouldn't there be a `@watch()` setup for `active` incase the prop is added after initialization?

Reply:
Yerp. I'll add that in - and probably update docs to add that in add in addition to "call `open` on your alert" you can just "set active attribute on it".

Reply:
Maybe we don't need the `open` / `close` methods at all anymore?

Reply:
So, we programmatically add `active` via the `open()` method, we could handle that in a smarter way through a `@watch` but, I'm curious if we just want to handle this case of active at load and try to recommend opening with `open()` - OR - remove these methods altogether as public methods, move them private, and just use them internally while telling folks to use `active` (the weird part has to do with queuing and setting active on multiple items to be queued, since it'd immediately get removed from the DOM but stored in the alert queue to launch)

Reply:
I would be very interested in removing the `open` and `close` methods on modal as well and moving everything to handle show/hide based on `active` prop. @macandcheese let me know if you figure this out.